packages/terra-dropdown-button/src/Item.jsx

```diff
@@ -44,16 +44,18 @@ const Item = ({
     navigate outside the dropdown with the dropdown still open if nothing else is pressed after opening the menu.
   */
   return (
-    <li tabIndex="-1" role="menuitem">
+    <li
```

Comment:
is this the a11y change you have alluded to in the pr description? is axe complaining that we're making a div interactable?

Reply:
so originally the change was to just drop the `role="menuitem"` from the `<div>`. This fixed the axe issue we were getting but the linter complained with the following errors.

-  error Static HTML elements with event handlers require a role jsx-a11y/no-static-element-interactions
- error `tabIndex` should only be declared on interactive elements jsx-a11y/no-noninteractive-tabindex

So I had to slightly change how we put together the `Item` component in general without changing functionality. With these new changes I made, I should have kept component functionality, how screen readers read it, and gotten rid of the errors we were were seeing in with axe.

Reply:
sounds good - just thinking, it would have been nice to have this rationale initially on an issue, or the pr description, so we have context on what changes we are making, and why